recipes/_install.rb

```diff
@@ -73,6 +73,13 @@ def nsp
   mode '0700'
 end
 
+# SPL-89640 On upgrades, the permissions of this directory is set incorrectly.
```

Comment:
As too permissive or not permissive enough? (could be a slightly better comment if it answers this question).